### ISSUE TITLE
Invalid timestamp displayed as swarm error

### ISSUE THREAD

**Issue by Skelaton4** [loki-project/loki-messenger#342]
I can see incoming friend request messages but I cannot reply or start my own friend requests. This is the error I receive. I am on Windows 10.  

![image](https://user-images.githubusercontent.com/28955674/60891910-3603f300-a22c-11e9-8817-eb45ea7844ec.png)

**Comment by BeaudanBrown** [comment]
Sorry for only just getting back to this, the testnet has been a bit unstable as we have been taking down nodes and testing new binary revisions for lokid and the storage server quite frequently
But having messages fail to this many contacts (i.e. they are likely to be spread across several swarms) is a bit concerning
If you were able to post the link to your logs by going `View -> Debug Log -> Submit` that would be really helpful, thanks

**Comment by Skelaton4** [comment]
I will run the debug tommorow and post it. Thanks!

**Comment by Skelaton4** [comment]
https://debuglogs.org/aa004948446294426ebb6ee054599616c393a3836959e54be1c3428ef31a18a2

**Comment by BeaudanBrown** [comment]
(First just a quick point for the future, if you press submit on the debug log form it will upload them and give you a link you can post instead of the whole thing)
But the important stuff here is this:
```
WARN 2019-07-15T11:45:59.640Z Loki send message: {"name":"HTTPError","message":"406 Error: Loki_rpc error","response":{"size":0,"timeout":10000}}
WARN 2019-07-15T11:45:59.640Z HTTPError body: Timestamp error: check your clock
```
So either your computer clock is out of sync somehow, or possibly there is a bug in our code for *checking* if the timestamp is valid
Are you able to look for anything weird going on with your computer clock?
We should probably look into displaying this error to the user somehow in the future...

**Comment by Skelaton4** [comment]
> (First just a quick point for the future, if you press submit on the debug log form it will upload them and give you a link you can post instead of the whole thing)
> But the important stuff here is this:
> 
> ```
> WARN 2019-07-15T11:45:59.640Z Loki send message: {"name":"HTTPError","message":"406 Error: Loki_rpc error","response":{"size":0,"timeout":10000}}
> WARN 2019-07-15T11:45:59.640Z HTTPError body: Timestamp error: check your clock
> ```
> 
> So either your computer clock is out of sync somehow, or possibly there is a bug in our code for _checking_ if the timestamp is valid
> Are you able to look for anything weird going on with your computer clock?
> We should probably look into displaying this error to the user somehow in the future...

Thank you for the tip! Now I know. I did not have my computer clocked synced to update automatically, after making the change the messages worked perfectly. I would recommend something that prompts the user if the clock is not in sync, that would help avoid this issue in the future. 

Thanks a lot for the help!!

**Comment by BeaudanBrown** [comment]
Going to keep open as reminder to show the timestamp error to user

**Comment by BeaudanBrown** [comment]
Fixed by #373
